shared/middleware/files_api.rb

```diff
@@ -606,7 +606,7 @@ def copy_file(endpoint, encrypted_channel_id, filename, source_filename)
     content_type :json
 
     filename.downcase! if endpoint == 'files'
-    get_bucket_impl(endpoint).new.list_versions(encrypted_channel_id, filename).to_json
+    get_bucket_impl(endpoint).new.list_versions(encrypted_channel_id, filename, with_comments: request.GET['with_comments']).to_json
```

Comment:
Not a comment for your work, but what is the deal with needing all caps GET to access a param? Seems odd to me

Reply:
I have no idea, especially as it seems to work without the `.GET`. I'm going to keep this as is to be inline with the rest of the file, unless you object.

Reply:
haha weird! yeah totally fine, was just observing